sort.php

```diff
@@ -66,6 +66,10 @@ function bubbleSort($sort_array, $checkEquality)
     $makeUnique = array_values(array_unique($makeUnique));
 }
 
+if (key_exists("b", $options)) {
+    $makeUnique = array_map("ltrim", $makeUnique);
+}
+
```

Comment:
A bit too late.

Reply:
I will then have to call ``file ()`` twice.

Reply:
You don't even need file() here.